### PR TITLE
td check in ui

### PR DESCRIPTION
**Pull request by timdavila**
https://trello.com/c/04V7Xdxu/5990-web-check-in-phase-2

### CODE REVIEW

File: CmsData/CmsData.csproj [hunk 1]
```diff
@@ -820,6 +820,11 @@
     <Compile Include="Generated\Views\TransactionSummary.cs" />
     <Compile Include="DbUtil\ITableWithExtraValues.cs" />
     <Content Include="App_Readme\Elmah.txt" />
+    <Content Include="Migrations\20191216_add_people_qr_data.sql" />
+    <Content Include="Migrations\20191216_pending_check_in_updates.sql" />
+    <Content Include="Migrations\20191212_add_earlylate_checkin_settings.sql" />
+    <Content Include="Migrations\20191209_check_in_profile_updates.sql" />
```
Comment (brondavies) [inline CmsData/CmsData.csproj]: I don't want to add these into the project file anymore because it just results in a lot of git conflicts.  Keep the files and remove these lines.

File: CmsData/Generated/Family.cs [hunk 2]
```diff
@@ -1,999 +1,1160 @@
...
-
-        private EntityRef<Person> _HeadOfHouseholdSpouse;
-
-        private EntityRef<Picture> _Picture;
-
-        private EntityRef<ResidentCode> _ResidentCode;
+	[Table(Name="dbo.Families")]
+	public partial class Family : INotifyPropertyChanging, INotifyPropertyChanged
+	{
+		private static PropertyChangingEventArgs emptyChangingEventArgs = new PropertyChangingEventArgs(String.Empty);
```
Comment (brondavies) [inline CmsData/Generated/Family.cs]: undo this change

File: CmsWeb/Areas/Public/Models/CheckInAPIv2/Results/AddEditPersonResults.cs [hunk 3]
```diff
@@ -8,5 +9,6 @@ public class AddEditPersonResults
 		public int familyID = 0;
 		public int peopleID = 0;
 		public int positionID = 0;
+        public Guid barcodeID = Guid.Empty;
```
Comment (brondavies) [inline CmsWeb/Areas/Public/Models/CheckInAPIv2/Results/AddEditPersonResults.cs]: Is this actually used?

File: CmsWeb/Areas/Public/Controllers/CheckInAPIv2Controller.cs [hunk 4]
```diff
@@ -478,6 +591,7 @@ public ActionResult PrintJobs( string data )
 
 			List<PrintJob> printJobs = (from label in CurrentDatabase.PrintJobs
 												where kiosks.Contains( label.Id )
+                                                where label.JsonData.HasValue()
```
Comment (brondavies) [inline CmsWeb/Areas/Public/Controllers/CheckInAPIv2Controller.cs]: You can't use HasValue() in a linq expression because it's an extension method that linq doesn't understand.  Instead use `label.JsonData != null && label.JsonData != ""`

File: CmsWeb/Areas/Public/Models/CheckInAPIv2/Results/AddEditPersonResults.cs (hunk 3 above)
Comment (timdavila) [reply CmsWeb/Areas/Public/Models/CheckInAPIv2/Results/AddEditPersonResults.cs]: yes, to reload the last search, the barcode is the most reliable method.

File: CmsData/CmsData.csproj (hunk 1 above)
Comment (timdavila) [reply CmsData/CmsData.csproj]: 👍

File: CmsData/Generated/Family.cs (hunk 2 above)
Comment (timdavila) [reply CmsData/Generated/Family.cs]: the whole file? I think all I changed is add a FamilyId FK on the PendingCheckIn, which should create a reference for PendingCheckIns for Family. but dbmlbuilder did a lot of other reformatting.

Comment (brondavies) [reply CmsData/Generated/Family.cs]: no, just this line
